xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/extension.vm

```diff
@@ -576,6 +576,14 @@ $namespace##
   </dd>
 #end
 
+#macro (spinner)
```

Comment:
Duplicate of 
- https://github.com/xwiki/xwiki-platform/blob/be3b9476e9bb02ccccbd1edeaf8e7bf6d058e916/xwiki-platform-core/xwiki-platform-export/xwiki-platform-export-pdf/xwiki-platform-export-pdf-ui/src/main/resources/XWiki/PDFExport/AdminSection.xml#L90
- https://github.com/xwiki/xwiki-platform/blob/466b48df3d2dab96b1bffda9e6fce90be848a4bf/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/extension.vm#L579

Reply:
:+1: 

TODO provide this spinner as a macro in https://github.com/xwiki/xwiki-platform/blob/29111f02b38e529f44ff88aaf4ec67af92fcf3df/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm (which from my experience is the main place where we provide API-ish velocimacros).